### PR TITLE
Documentation review: evolution.md

### PR DESCRIPTION
**Pull request by malloryfreeberg**
- Updated `evolution.md` document in preparation for the closed beta release
- Moved "Adding new committers" section to `committers.md`
- Ignore `jsonBrowser/` updates

### CODE REVIEW

File: docs/evolution.md [hunk 1]
```diff
@@ -3,102 +3,86 @@
...
-
-## Adding new committers to metadata working group
-
-A nominated group of metadata working group members will have commit access to the metadata-schema repo.  Committers are members of the working group who have permission to merge and accept pull requests to the metadata-schema repo.
-
-Committers are not allowed to skip the review process. Any person including the committers who wish to make a change to a metadata schema module this process must be followed.
-
-Anyone can request to join the committer team. When a request is received, the existing committers will discuss the request and vote on the addition of the new member. This process should be rapid and the requestor should hear the outcome within 72 hours.
+The Metadata Working Group should review the stability level of schemas on a quarterly or bi-annual basis to decide if changes are needed. For example, a new, low stability schema might become standard over time and be promoted to medium or even high stability once it has been reviewed and agreed upon by the HCA community.
```
Comment (malloryfreeberg) [inline docs/evolution.md]: @lauraclarke @claymfischer Do we still want to do this? We haven't ever AFAIK. If we don't plan to do this, we should remove this section.

File: docs/evolution.md [hunk 2]
```diff
@@ -3,102 +3,86 @@
...
 
 ## Governance of schema updates
 
-The Metadata Working Group will report progress to the HCA executive committee. The Metadata Working Group will review the update process on a yearly basis and ensure it is meeting the needs of the working group. If at any point the process becomes problematic, changes will be made to ensure metadata update is not a blocker for the consortium as a whole. The working group will provide written updates on the following matters at the start of each month:
+The metadata working group will review the update process on a yearly basis and ensure it is meeting the needs of the group. If the process becomes problematic, changes will be made to ensure metadata updates are not a blocker for the consortium as a whole. The working group will provide written updates on the following matters at the start of each quarter:
```
Comment (malloryfreeberg) [inline docs/evolution.md]: @lauraclarke @claymfischer Do we still want to do this? We haven't ever AFAIK. If we don't plan to do this, we should remove this section.

File: docs/evolution.md [hunk 3]
```diff
@@ -3,103 +3,87 @@
...
 - [Introduction](#introduction)
 - [Schema versioning](#schema-versioning)
-- [Module stability](#module-stability)
-- [Adding new committers to metadata working group](#adding-new-committers-to-metadata-working-group)
+- [Schema stability](#schema-stability)
 - [Governance of schema updates](#governance-of-schema-updates)
 
 ## Introduction
 
 This document describes the principles and standards by which the HCA metadata schema will evolve as well as the semantics for versioning and updating the schema. The HCA is collecting data from complex biological samples and assays with rich metadata. We expect over the lifetime of the project that the schemas that captures these metadata will need to change. These changes will always be to support the main goal of the HCA Data Coordination Platform (DCP): enabling downstream use and interpretation of the data. As our understanding of the data changes, the metadata we need may also change. The schemas, therefore, will also need to evolve to support new assays and changing practices in the contributing labs as the precise steps conducted for a particular assay are improved.
```
Comment (daniwelter) [inline docs/evolution.md]: Typo: [...] the schemas that capture[...] - not captures

File: docs/evolution.md [hunk 4]
```diff
@@ -3,103 +3,87 @@
...
+- Adding a regular expression to a field.
 
-**Minor (increment to Y)**: Any change to the schema that ensures compatibility with metadata created using the previous version. Specific examples include:
+**Minor (increment to Y)**: Any schema change that ensures backwards compatibility with metadata created using the previous version.
 - Adding an optional field.
 - Making a required field optional.
-- Updating the field description in a way that changes the meaning of the field.
-- Adding a new module (increment the type schema that references the new module).
-- Removing controlled vocabulary from a field (but the field type stays the same).
+- Removing a controlled vocabulary from a field (but the field type stays the same).
```
Comment (daniwelter) [inline docs/evolution.md]: I find this line confusing - I had to check the previous section to make sure that included the removal of specific values. Could we rephrase to "Changing a field from controlled vocabulary to free text with the field type staying the same" or something like that?

File: docs/evolution.md [hunk 5]
```diff
@@ -3,103 +3,87 @@
...
-**Patch (increment to Z)**: A change to auxiliary information that does affect the structure or semantics of the schema itself. Specific examples include:
+**Patch (increment to Z)**: Any change to auxiliary information that does not affect the structure or semantics of the schema itself. Also, any bug fixes to the schema.
 - Adding or updating the field description to improve readability.
-- Adding or updating the field example.
-- Adding or updating the field comment.
-- Adding or updating the field user-friendly name.
+- Adding or updating the field example, comment, or user-friendly name.
 - Extending a controlled vocabulary enumeration.
-- Fixing an incorrect regular expression.
+- Correcting a regular expression.
```
Comment (daniwelter) [inline docs/evolution.md]: I would add "incorrect" to both of these. I know correcting an incorrect sounds unnecessarily precise but people misuse "correct" when they mean "change" so I want to make it really clear that the reason for the change is that the old value was wrong or broken.

File: docs/evolution.md [hunk 6]
```diff
@@ -3,103 +3,87 @@
...
+It is important to note that the stability of a schema is not the same as the schema version, although the two ideas are related. There are no restrictions on the type of update (major, minor, patch) that can happen to a schema based on its stability; however, the consequences of a major update on a high stability schema are much more severe than a patch update to a low stability schema, for example. Both schema stability and update type should be considered when proposing a schema update.
```
Comment (daniwelter) [inline docs/evolution.md]: Apples & oranges - consider changing to a more related comparison, ie [...] much more severe than a major update to a low stability schema[...]

File: docs/evolution.md [hunk 7]
```diff
@@ -3,103 +3,87 @@
...
+The metadata team should review the stability level of schemas on a quarterly or bi-annual basis to decide if changes are needed. For example, a new, low stability schema might become standard over time and be promoted to medium or even high stability once it has been reviewed and agreed upon by the HCA community.
 
 ## Governance of schema updates
 
-The Metadata Working Group will report progress to the HCA executive committee. The Metadata Working Group will review the update process on a yearly basis and ensure it is meeting the needs of the working group. If at any point the process becomes problematic, changes will be made to ensure metadata update is not a blocker for the consortium as a whole. The working group will provide written updates on the following matters at the start of each month:
+The metadata team will review the update process on a yearly basis and ensure it is meeting the needs of the group. If the process becomes problematic, changes will be made to ensure metadata updates are not a blocker for the consortium as a whole. The metadata team will provide written updates on the following matters at the start of each quarter:
```
Comment (daniwelter) [inline docs/evolution.md]: will provide written updates where?

File: docs/evolution.md [hunk 8]
```diff
@@ -3,103 +3,87 @@
...
 
-- Changes to the metadata specification
-  - Updates to existing modules
-  - Addition of new modules
-- New members of the metadata working group
-- New committers to the metadata-schema git repo
+- Changes to the metadata standard and specification
+  - Updates to existing schemas
+  - Addition of new schemasS
```
Comment (daniwelter) [inline docs/evolution.md]: extraneous S

File: docs/evolution.md (hunk 5 above)
Comment (malloryfreeberg) [reply docs/evolution.md]: What about "Fixing an incorrect..." to avoid double-correct

**Comment by malloryfreeberg** [comment]
@daniwelter made the updates!

**Pull request merged**